Services/AssessmentQuestion/interfaces/interface.ilAsqQuestionAuthoring.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/* Copyright (c) 1998-2013 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * Interface ilAsqQuestionAuthoring
+ *
+ * @author    Björn Heyser <info@bjoernheyser.de>
+ * @version    $Id$
+ *
+ * @package    Services/AssessmentQuestion
+ */
+interface ilAsqQuestionAuthoring
```

Comment:
This could have the GUI-prefix IMO.

Reply:
+1

Reply:
interfaces with gui prefix? Of course the iplementing classes need to be named with gui according to the requirements of the ilias control structure. But i can change of course.